setup-signify.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euo pipefail 
+
+sudo apt -y install signify-openbsd qrencode
+
+# clear out old keys
+rm -f key.pub key.sec
```

Comment:
should this be `/vx/config/key.pub` etc.?

Reply:
oh whoops, yeah.